### DISCUSSION TITLE
Importing Data

### DISCUSSION THREAD

**Discussion by egohookx** [beginner-corp/begin-community#83] [Q&A] [answered]
Wondering how best to import some data that should persist across deployments. I can't seem to wrap my head around how that works. Is there a console or some other method to import a table and have my app interact with it, or is it a matter of importing the data via code in my dev repo, and trusting that this will replicate when pushing to staging (and later production)?

Thanks in advance.

**Comment by pchinjr** [comment, marked as answer]
Hi there, you can use a startup script locally to populate your local environment with a `prefs.arc` file: https://arc.codes/docs/en/reference/prefs.arc/sandbox-startup. This question does come up quite a bit, so you're not the only person wondering. 
For your testing and production environments, we don't have a feature to seed it automatically. You would have to create your own POST requests that you could script to inject the data into your 'staging' and 'production' environments. 
We're so sorry this feature is missing right now, being able to have CRUD access to your Begin Data table is on the roadmap and we're giving it plenty of thought.

**Reply by egohookx** [reply]
Thanks for the response. A CRUD interface for this data would indeed be great. I know that roadmapping things like this is hard. I'm sure there are many other priorities to consider. Perhaps a good interim solution would be a way to simply allow the upload/entry of JSON that turns into tables, with perhaps the option to simply delete tables as necessary. Keep up the interesting work.
